global.R

```diff
@@ -124,59 +124,93 @@ get_data <- function(data_type = 'census') {
       # Address the weirdness with "New Credit (Part)"
       temp_data$Geography <- gsub('(Part) ', '', temp_data$Geography, fixed = TRUE)
       # Keep only the first part of the name (what is with the %?)
-      temp_data$Geography <- paste0(unlist(lapply(strsplit(temp_data$Geography, ')', fixed = TRUE), function(x){x[1]})), ')')
+      temp_data$Geography <- paste0(unlist(lapply(strsplit(temp_data$Geography, ')', fixed = TRUE),
+                                                  function(x){x[1]})), ')')
+
+      # give Ontario four digit number to subset by.
+      temp_data$Geography <- ifelse(grepl('Ontario', temp_data$Geography), 'Ontario', temp_data$Geography)
+
       #subset to Ontarios and 4 digit geo codes
       geo_codes <- unlist(lapply(strsplit(temp_data$Geography,
-                                          '(', fixed = TRUE), 
+                                          '(', fixed = TRUE),
                                  function(x){
                                    gsub(')', '', x[2], fixed = TRUE)}))
-      # (those wih NA for the geo_code are all ontario)
+
+      # (those wih NA for the geo_code are all ontario) - give it 3500 so we can subset
+      # entirely by 4 digit geo_code
       temp_data$geo_code <- geo_codes
-      temp_data$geo_code[is.na(temp_data$geo_code)] <- 'Ontario'
+      temp_data$geo_code[is.na(temp_data$geo_code)] <- '3500'
+
+      # keep only rows that have 4 number
```

Comment:
@benmbrew What are the rows that don't have 4 numbers? I recall many having more than 4 - what does that mean?